README.md

```diff
@@ -5,3 +5,5 @@ that help you build apps, integrate services, and connect to your Atlas data fas
 
 Currently the Atlas App Services examples just contains examples of [Atlas Triggers](https://www.mongodb.com/docs/atlas/app-services/triggers/),
 which you can find in the [triggers-examples folder](./triggers-examples).
+
+For example projects featuring Atlas App Services with Atlas Device SDK, see the `Atlas Device SDK Example Projects page <https://www.mongodb.com/docs/realm/example-projects/>`.
```

Comment:
I wonder if we want to give a little more info about what devs will find if they go to this page? Maybe something like:

```suggestion
For a wide range of example projects showing common development patterns and use cases, see the `Atlas Device SDK Example Projects page <https://www.mongodb.com/docs/realm/example-projects/>`.
```